src/components/icon/index.d.ts

```diff
@@ -9,6 +9,8 @@ interface IconProps {
   text?: Boolean;
 }
 
-declare const Icon: BulmaComponentWithoutRenderAs<IconProps, 'span'>;
+declare const Icon: BulmaComponentWithoutRenderAs<IconProps, 'span'> & {
```

Comment:
I realized using `BulmaComponentWithoutRenderAs` is wrong since `Icon` clearly accepts a `renderAs` prop. My bad.